source/includes/usage-examples/code-snippets/distinct.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	var uri string
+	if uri = os.Getenv("DRIVER_REF_URI"); uri == "" {
+		log.Fatal("You must set your 'MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
+	}
+
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin distinct
+	coll := client.Database("sample_mflix").Collection("movies")
+	query := bson.D{{"directors", "Natalie Portman"}}
```

Comment:
The [Go driver API docs](https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo#Collection.Distinct) describes this as a filter

```suggestion
	filter := bson.D{{"directors", "Natalie Portman"}}
```